trailblazer/store/api.py

```diff
@@ -269,7 +272,7 @@ def query_slurm(job_id_file: str, case_id: str, ssh: bool) -> Any:
     @staticmethod
     def get_time_elapsed_in_min(elapsed_string: Optional[str]) -> Optional[int]:
         """Parse SLURM elapsed time string into minutes"""
-        if elapsed_string:
+        if elapsed_string and isinstance(elapsed_string, str):
```

Comment:
why is this needed?

Reply:
there could be a case when the value is <float> = nan, whcih isnt == None





10:01
men bara om man får trunkerat svar